### DISCUSSION TITLE
Is there an api which I can use to update the existing license list in DT server?

### DISCUSSION THREAD

**Discussion by swatissm** [DependencyTrack/dependency-track#1500] [Q&A]
Is there an api which I can use to update the existing license list in DT server?

As need to check if I want to add more licenses to the list of Dependency Track instance I am running ? Is there an API or UI to do that ?

**Comment by stevespringett** [comment]
No, there isn't a way to do this. Currently, DT ships with SPDX license list v3.11. DT v4.5 will include SPDX license list v3.16 (or higher if available).
